scripts/build/commands/all.js

```diff
@@ -0,0 +1,28 @@
+const path = require(`path`);
+const {
+  webpackBuild,
+  transpile
+} = require(`../../utils/build`);
+const {getAllPackagePaths} = require(`../../utils/package`);
+
+
+module.exports = {
+  command: `all`,
+  desc: `Build all packages`,
+  builder: {},
+  handler: () =>
+    getAllPackagePaths().map((pkg) => {
```

Comment:
This is going to run each package in parallel. when lerna did stuff like that, it tried to melt my cpu; should these be more sequential?